components/user/userApi.ts

```diff
@@ -76,6 +85,22 @@ export const userApi = createApi({
       }),
       invalidatesTags: ['FriendRequest'],
     }),
+    getGlobalLeaderboard: builder.query<RLeaderboard, QLeaderboard>({
+      query: (paginationParams) => ({
+        url: '/leaderboard/global/xp',
+        method: 'GET',
+        params: paginationParams,
+      }),
+      providesTags: ['Leaderboard'],
```

Comment:
Do we invalıdate these in any other place? If not, we don't need to provide these tags.